react/backstop/backstop.js

```diff
@@ -3,11 +3,14 @@ const path = require('path');
 const listDirs = require('./listDirs.js');
 const storyBookBackstop = require('./storyBookBackstop');
 
-const processAtoms = (process.argv.indexOf('--atoms') > -1);
 const debug = (process.argv.indexOf('--debug') > -1);
 
 // The value of viewports will depend on whether the --atoms switch was set.
-let viewports;
+const viewports = [
+  { label: 'small_atom', width: 400, height: 250 },
+  { label: 'phone', width: 320, height: 480 },
+  { label: 'tablet', width: 1024, height: 768 }
+];
```

Comment:
Should this be an exported const so we're not duplicating these values between ``backstop.js`` and ``storyBookBackstop.js`` ?